src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java

```diff
@@ -237,6 +237,12 @@ public class MiscConfig {
     @Expose
     public int limboTimePB = 0;
 
+    @Expose
+    public int limboPlaytime = 0;
```

Comment:
Those values belong in the storage, not in the config. also move limboTimePB into storage (and hopefully migration also works here)